Update_user.css

```diff
@@ -72,44 +111,108 @@ nav a{
 	height:680px;
 	width:100%;
 }
-
+/* Set up the footer */
```

Comment:
I update the footer with all the element that I will decided to add. Like the fonts, color